packages/loaders/src/index.js

```diff
@@ -1,5 +1,10 @@
-import { Application } from '@pixi/app';
-import Loader from './loader';
+/**
+ * Reference to **resource-loader**'s Resource class.
+ * See https://github.com/englercj/resource-loader
```

Comment:
Might be more helpful to link to the Resource docs directly? http://englercj.github.io/resource-loader/Resource.html